data-processing/requirements.txt

```diff
@@ -9,6 +9,7 @@ pysbd
 reportlab
 requests
 TexSoup==0.2.0
+nltk
```

Comment:
Clarification question: will things go wrong if another more recent version of nltk is installed? If so, then perhaps a version number should be specified here

Reply:
Good point! Added version info. Though, I only use `nltk` and `numpy` in definition extractor now.